File: message-api-schema-validator/src/main/java/uk/ac/jisc/rdss/tests/vocabulary/create/VocabularyCreateRequestTest.java
package uk.ac.jisc.rdss.tests.vocabulary.create;

import org.junit.Test;

import uk.ac.jisc.rdss.tests.AbstractSchemaValidatorTest;

public class VocabularyCreateRequestTest extends AbstractSchemaValidatorTest {

    @Override
    protected String getSchemaFileName() {
        return "messages/vocabulary/create/request_schema.json";
    }

    @Test
    public void validateRequest() throws Exception {
        validateJson("messages/vocabulary/create/request.json");
    }
}

File: message-api-schema-validator/tests/vocabulary/vocabulary_create_request_test.py
from tests.abstract_schema_validator_test import AbstractSchemaValidatorTest
from unittest import TestCase


class VocabularyCreateRequestTest(AbstractSchemaValidatorTest, TestCase):
    def get_json_schema_file_name(self):
        return "messages/vocabulary/create/request_schema.json"

    def test_validate_request(self):
        self.validate_json("messages/vocabulary/create/request.json")
